tests/Autotest-Uint-SafeMath.test.ts

```diff
@@ -0,0 +1,63 @@
+import {DivisionByZeroError} from "../lib/errors";
+import {assertErrorMsg, validateUint} from "./helpers";
+import {getUints} from "./Autotest-config";
+
+getUints().forEach((tc) => {
+    describe(`SafeMath tests for ${tc.name}`, () => {
+        it(`should return a ${tc.name} for all basic methods: Add`, () => {
+            const one: any = tc.constructor("1");
```

Comment:
Could we replace type `any` with `Uint`?

Reply:
Typescript wasn't happy with the Uint/Int super-types. I think that was my initial intent behind #9, having identical constructors I think resolves this, will pull from master and see if it lets me.

Reply:
I get the error `TS2349: Cannot invoke an expression whose type lacks a call signature` when using Int/Uint. Any ideas? @JonathanLorimer?

Reply:
After many attempts I think I'm going to propose we put this off for now. This seems to be an additional feature and does not impose any immediate limitations as far as I can see. I will open an issue to address it and we can revisit it with any bright ideas that may come about.